Userland/Applications/Browser/InspectorWidget.cpp

```diff
@@ -196,6 +203,12 @@ void InspectorWidget::update_node_box_model(StringView node_box_sizing_json)
     m_element_size_view->set_box_model(m_node_box_sizing);
 }
 
+void InspectorWidget::update_aria_properties_state_model(StringView aria_properties_state_json)
+{
+    m_aria_properties_state_view->set_model(MUST(WebView::AriaPropertiesStateModel::create(aria_properties_state_json)));
```

Comment:
Use `.release_value_but_fixme_should_propagate_errors()` instead of `MUST()` when an error might happen but we're not yet handling it. `MUST()` should only be used for things that are either logically impossible to fail, or we want to crash if they do. (Like a `VERIFY()` basically.)